cluster-autoscaler/core/scaleup/orchestrator/orchestrator.go

```diff
@@ -316,6 +279,21 @@ func (o *ScaleUpOrchestrator) ScaleUp(
 	}, nil
 }
 
+func (o *ScaleUpOrchestrator) applyLimits(newNodes int, resourcesLeft resource.Limits, nodeGroup cloudprovider.NodeGroup, nodeInfos map[string]*schedulerframework.NodeInfo) (int, errors.AutoscalerError) {
+	nodeInfo, found := nodeInfos[nodeGroup.Id()]
+	if !found {
+		// This should never happen, as we already should have retrieved nodeInfo for any considered nodegroup.
+		klog.Errorf("No node info for: %s", nodeGroup.Id())
+		return 0, errors.NewAutoscalerError(errors.CloudProviderError, "No node info for best expansion option!")
+	}
+
+	limittedNewNodes, aErr := o.resourceManager.ApplyLimits(o.autoscalingContext, newNodes, resourcesLeft, nodeInfo, nodeGroup)
```

Comment:
nit: `limitted` -> `limited`. Alternatively, why not just `return o.resourceManager.ApplyLimits(...)` directly?

Reply:
Good point, changed it to returning the result directly.